context/types.ts

```diff
@@ -19,6 +19,11 @@ export type GlobalProviderProps = {
   children: ReactNode
 }
 
+export type MoveItemAction = {
+  payload: any
```

Comment:
Is it possible to be more specific here? What are the different types payload can be?